src/components/material/material-buttons/online/MaterialButtonOnlineExternal.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { FC } from "react";
+import { useText } from "../../../../core/utils/text";
+import { LinkNoStyle } from "../../../atoms/link-no-style";
+import { Button } from "../../../Buttons/Button";
+
+export interface MaterialButtonOnlineExternalProps {
+  externalUrl: string;
+  origin: string;
+}
+
+const MaterialButtonOnlineExternal: FC<MaterialButtonOnlineExternalProps> = ({
+  externalUrl = "https://google.com",
```

Comment:
why google.com here?

Reply:
I just thought that until we have some concrete external urls I would use google. It was the most generic url I could think of.